spec/features/save_subquestion_clash_spec.rb

```diff
@@ -13,9 +13,13 @@
 
     answer = Quby.send(:answer_repo).send(:all_records, 'subquestion_key_clash').last
 
-    expect(answer.raw_params).to include("aborted" => false,
-                                         "v_0" => "a1",
-                                         "v_0_a1" => "clashing")
+    params = Rails.version >= '5' ? answer.raw_params.to_unsafe_h : answer.raw_params
```

Comment:
voelt ook niet lekker. Dit komt uit de memory store neem ik aan? Moet dat de memory store niet worden aangepast? Of mogelijk op een hoger niveau? Gaat opslaan in db store dan wel goed?